File: typescript/src/get_vehicles.ts
import { EmptySchema } from "@bufbuild/protobuf/wkt";
import { create, toJsonString } from "@bufbuild/protobuf";
import { GetVehiclesResponseSchema } from "@buf/nativeconnect_api.bufbuild_es/nativeconnect/api/v1/nativeconnect_pb";

import { createNodeClient } from "./client"

const client = createNodeClient()

const request = create(EmptySchema)

const response = await client.getVehicles(request)

console.log(toJsonString(GetVehiclesResponseSchema, response, { prettySpaces: 2 }))

File: python/get_vehicles.py
from client import create_native_client
from google.protobuf.empty_pb2 import Empty


def main():
    client = create_native_client()
    request = Empty()
    response = client.GetVehicles(request)
    print(response)

     
if __name__ == "__main__":
    main()
